data/news.ts

```diff
@@ -11,18 +11,10 @@ export const newsItems: NewsInterface[] = [
   },
   {
     type: "learn",
-    title: "Revisiting Nova. How to handle cycles of curves within folding schemes by Wilson Nguyen. Aug/9, 16:00 UTC",
+    title: "Private markets on Ethereum - 0xParc Piere. August 23, 2023 14:00 (UTC)",
```

Comment:
1. Small detail: I think they mostly write it as "0xPARC"
2. When I click the link It says 14:00 CEST, so I assume it must be 12:00 (UTC)?

Reply:
it's an error on discord event, i just fixed the time.